skills_taxonomy_v2/pipeline/skills_extraction/skills_naming_utils.py

```diff
@@ -88,81 +123,44 @@ def clean_cluster_description(sentences):
         cluster_descriptions (dict): Cluster number : list of cleaned
             sentences in this cluster
     """
-    # Init the Wordnet Lemmatizer
-    lemmatizer = WordNetLemmatizer()
 
     # How many times a n-gram has to occur in order for occurences of them
     # to be converted to a single dash separated word
     num_times_ngrams_thresh = 3
 
-    work_stopwords = [
-        "essential",
-        "requirement",
-        "required" "degree",
-        "responsibility",
-        "duties",
-        "responsibilities",
-        "experienced",
-        "previous",
-        "andor",
-        "minimum",
-        "year",
-        "skill",
-        "ideal",
-        "candidate",
-        "desirable",
-        "willing",
-        "prepared",
-        "knowledge",
-        "experience",
-        "skills",
-        "ideally",
-        "responsible",
-        "require",
-        "environment",
-        "role",
-        "work",
-        "job",
-        "description",
-        "ymy",
-    ]
-
-    all_stopwords = stopwords.words("english") + work_stopwords
-
-    cluster_docs_cleaned = []
+    # Don't append duplicates, just use set
+    cluster_docs_cleaned = set()
```

Comment:
when I ran this, I got a `list object has no attribute add` error - I changed it back to cluster_docs_cleaned = list ()

Reply:
how very strange! it's clearly defined as a set object, so no idea why it's coming up saying it's a list object! Also you think it would have errored when I ran it too. I feel like something odd is going on here 😕  Anyway, should do the same thing anyway!

Reply:
yeah :/